services/dy-dash/mattward-dash/.cookiecutterrc

```diff
@@ -0,0 +1,37 @@
+# This file exists so you can easily regenerate your project.
+#
+# `cookiepatcher` is a convenient shim around `cookiecutter`
+# for regenerating projects (it will generate a .cookiecutterrc
+# automatically for any template). To use it:
+#
+#    pip install cookiepatcher
+#    cookiepatcher gh:itisfoundation/cookiecutter-simcore-pyservice project-path
+#
+# See:
+#    https://pypi.python.org/pypi/cookiepatcher
+#
+# Alternatively, you can run:
+#
+#    cookiecutter --overwrite-if-exists --config-file=project-path/.cookiecutterrc gh:itisfoundation/cookiecutter-osparc-service
+#
+
+default_context:
+
+    _extensions:               ['jinja2_time.TimeExtension']
+    _template:                 'gh:ITISFoundation/cookiecutter-osparc-service'
+    author_affiliation:        "IT'IS Foundation"
+    author_email:              'maiz@itis.swiss'
+    author_name:               'Odei Maiz'
+    contact_email:             'maiz@itis.swiss'
+    default_docker_registry:   'https://index.docker.io/v1'
+    git_username:              'odeimaiz'
+    number_of_inputs:          '0'
+    number_of_outputs:         '8'
+    project_git_repo:          'https://github.com/odeimaiz/osparc-services'
+    project_name:              'MattWard-Dash'
+    project_package_name:      'mattward-dash'
+    project_short_description: 'MattWard-Dash'
+    project_slug:              'mattward-dash'
+    project_type:              'computational'
+    release_date:              '2019'
+    version:                   '0.2.0'

```

Comment:
you'll probably have to go for version 1.0.0 for the first version (I think all are like that in the end)